data/teammembers.js

```diff
@@ -91,12 +93,27 @@ export default {
   ],
   cerritos: [
     {
-      name: 'Lucy Wan',
-      position: 'Vice President',
-      introduction: `What I like about Peer1on1: I like that Peer1on1 gives me the opportunity to be able to help and spend time with special needs people on a weekly basis. I also really enjoy the activities and I hope to be a positive influence to their lives.
-      \n My Future Goals: I am pursuing a career as an artificial intelligence scientist with applications in neuroscience.`,
-      avatar_img: LucyWan,
```

Comment:
I would like to suggest keeping all the inactive members instead of removing them, just set the flag "is_active" to "false", and then you can go to the file "src/components/TeamMembers.js" to change this code (line 44)
` 
members.map((member, i) => {
`
to
```
members.filter((member) => {
        return member.is_active === true;
        }).map((member, i) => {
```